crates/log-server/src/rocksdb_logstore/builder.rs

```diff
@@ -144,7 +145,7 @@ fn cf_data_options(
 
         opts.set_compression_per_level(&[
             DBCompressionType::None,
```

Comment:
The commit message reads as if we are enabling compression for L0/L1 files. Any reason to not enable it for L0 as we do it for the metadata cf?

Reply:
Good catch, it's a mismatch between a local change and what I sent in the PR, let me fix that.